docs/databases.rst

```diff
@@ -74,13 +74,13 @@ Quick Start
    from parsons import Postgres
 
    # Instantiate Postgres from environmental variables
-   mysql = Postgres()
+   pg = Postgres()
 
    # Instantiate Postgres from passed variables
-   Postgres = Postgres(username='me', password='secret', host='mydb.com', db='dev', port=3306)
+   pg = Postgres(username='me', password='secret', host='mydb.com', db='dev', port=3306)
 
    # Instantiate Postgres from a ~/.pgpass file
-   Postgres = Postgres()
+   pg = Postgres()
 
 **Quick Start**
 
```

Comment:
Can't seem to leave a comment on the section below, but just a couple lines down, on 90 and 94, there's two references to `postgres` which should probably be `pg` now.